### PR TITLE
MAGN-7532: Localization of Excel and Engine warnings

### PR DESCRIPTION
**Pull request by nguyen-binh-minh**
Similar to #4643 
@ke-yu PTAL

### CODE REVIEW

**Comment by ke-yu** [comment]
@nguyen-binh-minh , thanks for fixing it! LGTM.

**Comment by ke-yu** [comment]
@nguyen-binh-minh , we need to fix it on master as well...

File: src/Libraries/DSOffice/Excel.cs [hunk 1]
```diff
@@ -55,7 +55,7 @@ public static Application RegisterAndGetApp()
             // get excel, throw exception if it is not
             var officeType = Type.GetTypeFromProgID("Excel.Application");
             if (officeType == null)
-                throw new Exception("Excel is not installed.");
+                throw new Exception(Resources.ExcelNotInstalled);
```
Comment (sharadkjaiswal) [inline src/Libraries/DSOffice/Excel.cs]: Needs to be part of DSOffice resource dll.

**Comment by nguyen-binh-minh** [comment]
updated

File: src/Libraries/DSOffice/DSOffice.csproj [hunk 2]
```diff
@@ -80,6 +85,11 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="DSOfficeImages.resx" />
+    <EmbeddedResource Include="Properties\Resources.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
```
Comment (sharadkjaiswal) [inline src/Libraries/DSOffice/DSOffice.csproj]: Do you need to make the generator public?

**Pull request merged**